### PR TITLE
Hardcode exe returned for windows store envs

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: crates/pet-core/src/python_environment.rs [hunk 1]
```diff
@@ -358,6 +359,43 @@ impl PythonEnvironmentBuilder {
...
 }
 
+// Given a list of executables, return the one with the shortest path.
+// The shortest path is the most likely to be most user friendly.
+fn get_shortest_executable(
+    category: &PythonEnvironmentCategory,
+    exes: &Option<Vec<PathBuf>>,
+) -> Option<PathBuf> {
+    // For windows store, the exe should always be the one in the WindowsApps folder.
+    if *category == PythonEnvironmentCategory::WindowsStore {
```
Comment (DonJayamanne) [inline crates/pet-core/src/python_environment.rs:369]: Harcoded for windows store

File: crates/pet-windows-store/src/environments.rs [hunk 2]
```diff
@@ -63,14 +63,15 @@ impl PotentialPython {
                 } else {
                     None
                 })
-                .version(Some(self.version.clone()))
+                // We only have the partial version, no point returning bogus info.
+                // .version(Some(self.version.clone()))
```
Comment (DonJayamanne) [inline crates/pet-windows-store/src/environments.rs:67]: Another change to never return bogus information (partial version != valid data)
And Pet does best effort to get valid/accurate data, or nothing at all.

Comment (DonJayamanne) [reply crates/pet-windows-store/src/environments.rs:67]: @karthiknadig /cc

File: crates/pet-core/src/python_environment.rs (hunk 1 above)
Comment (DonJayamanne) [reply crates/pet-core/src/python_environment.rs:369]: @karthiknadig /cc

**Review by chrmarti — APPROVED** [review]

**Pull request merged**